### PR TITLE
Document migration from Self-serve to Enterprise and Free to Pro

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB tier (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB tier (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+* (Optional but recommended) Capture existing index and constraint definitions:
+** Run the cypher statement `SHOW CONSTRAINTS YIELD createStatement` and save result to a file, to use later in the process.
+** Run the cypher statement `SHOW INDEXES YIELD createStatement` and save result to a file to use later in the process.
+
+* (Optional but recommended) Drop the indexes and constraints.
+** Run the cypher statement `SHOW CONSTRAINTS YIELD name RETURN 'DROP CONSTRAINT ' + name + ';'` in order to generate the commands to drop existing constraints. 
+** Execute the generate commands to drop existing constraints.
+** Run the cypher statement `SHOW INDEXES YIELD name RETURN 'DROP INDEX ' + name +';'` in order to generate the commands to drop existing indexes. 
+* Then execute the generate commands to drop existing indexes.
+* For more detail regarding the capture and recreating of indexes and constrains, please refer to the link:https://support.neo4j.com/s/article/4416090515219-Indexes-Capture-Drop-Recreate-existing[Indexes - Capture / Drop / Recreate] knowledge base article.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:41]:
```suggestion
. (Optional but recommended) Capture existing index and constraint definitions:
.. Run the following Cypher statement:
+
[source,cypher]
----
SHOW CONSTRAINTS YIELD createStatement
----
+
Save result to a file, to use later in the process.
..
.. Run the following Cypher statement:
+
[source,cypher]
----
SHOW INDEXES YIELD createStatement
----
+
Save result to a file, to use later in the process.

. (Optional but recommended) Drop the indexes and constraints.
.. Run the following Cypher statement to generate the commands to drop existing constraints:
+
[source,cypher]
----
SHOW CONSTRAINTS YIELD name
RETURN 'DROP CONSTRAINT ' + name + ';'
----
.. Execute the generated commands to drop existing constraints.
.. Run the following Cypher statement to generate the commands to drop existing indexes:
+
[source,cypher]
----
SHOW INDEX YIELD name
RETURN 'DROP INDEX ' + name + ';'
----
.. Execute the generated commands to drop existing indexes.

For more information about indexes and constrains, see link:{neo4j-docs-base-uri}/cypher-manual/current/indexes/[Cypher Manual -> Indexes] and link:{neo4j-docs-base-uri}/cypher-manual/current/constraints/[Cypher Manual -> Constraints].
```

Comment (AlexicaWright) [reply modules/ROOT/pages/tutorials/migration-free.adoc:41]: Just like blogposts, we try not to link to KB articles.

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+=== In the console (Activities on AuraDB Free)
+
+* Download snapshot/Dump locally (the daily automatic snapshot)
+* In the Aura Console select the AuraDB Instance
+* Go to the *Snapshots* tab
+* Click the *three dots*, and select *Export*
+* Save locally the Dump file (preserve the .dump extension)
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:49]:
```suggestion
. Download snapshot/Dump locally (the daily automatic snapshot)
. In the Aura Console select the AuraDB Instance
. Go to the *Snapshots* tab
. Click the *three dots*, and select *Export*
. Save the dump file locally (preserve the .dump extension)
```

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+=== In the console (Activities on AuraDB Professional or AuraDB Enterprise)
+
+* Create a new AuraDB Instance in AuraDB Professional or AuraDB Enterprise with the right resource sizing.
+* Upload via Console drag and drop or push-to-cloud
+* From the Aura Console: link:https://neo4j.com/docs/aura/auradb/importing/importing-data/#_import_database[Drag and drop]
+* Using the command line: `neo4j-admin push-to-cloud`
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:56]:
```suggestion
. Create a new AuraDB Instance in AuraDB Professional or AuraDB Enterprise with the right resource sizing.
. Upload via Console drag and drop or push-to-cloud
.. From the Aura Console: link:https://neo4j.com/docs/aura/auradb/importing/importing-data/#_import_database[Drag and drop]
.. Using the command line: `neo4j-admin push-to-cloud`
```

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+* In the Aura Console select the AuraDB Instance
+* Go to the *Snapshots* tab
+* Click the *three dots*, and select *Export*
+* Save locally the Dump file (preserve the .dump extension)
+
+=== In the console (Activities on AuraDB Professional or AuraDB Enterprise)
+
+* Create a new AuraDB Instance in AuraDB Professional or AuraDB Enterprise with the right resource sizing.
+* Upload via Console drag and drop or push-to-cloud
+* From the Aura Console: link:https://neo4j.com/docs/aura/auradb/importing/importing-data/#_import_database[Drag and drop]
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:55]: Why is the link called _Drag and drop_?

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+* Using the command line: `neo4j-admin push-to-cloud`
+
+=== In the newly created AuraDB Professional or Enterprise Instance
+
+(Optional) Once the AuraDB Instance is loaded and started, you can now recreate the indexes and constraints, using the information captured earlier in the process or refer to link:https://support.neo4j.com/s/article/4416090515219-Indexes-Capture-Drop-Recreate-existing[Indexes - Capture / Drop / Recreate] knowledge base article.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:60]:
```suggestion
(Optional) Once the AuraDB Instance is loaded and started, you can recreate the indexes and constraints, using the information captured earlier in the process.
```
As above, avoid KB articles in the docs.

**Review by AlexicaWright — COMMENTED** [review]
Some comments.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+* Upgrade to Professional (Works for AuraDB)
+* Clone to New (Works for Aura Professional and AuraDS) 
+* Manual Process
+
+== Upgrade to Professional (Works for AuraDB)
+
+You can upgrade an instance to the Professional plan directly from the console.
+
+Click the *three dots* on an instance card > *Upgrade to Professional*
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:16]:
```suggestion
Click the ellipsis (*...*) button on an instance card > *Upgrade to Professional*
```

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+Verify that the cloud provider and region are correct and select the instance size you need.
+Note that the default version of Neo4j is 5.
+Once you are satisfied, click ***Upgrade***.
+
+== Clone (Works for Aura Professional and AuraDS)
+
+The other way is to clone your existing free tier AuraDB Instance to the Professional tier. 
+
+* Click the *three dots* on an instance
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:26]:
```suggestion
* Click the ellipsis (*...*) button on an instance 
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+Click the *three dots* on an instance card > *Upgrade to Professional*
+
+You will see "upgrade to professional". Note the warning regarding the default version is now version 5.
+
+== Clone (Works for Aura Professional and AuraDS)
+
+The other way is to clone your existing free tier AuraDB Instance to the Professional tier. 
+
+* Click the *three dots* on an instance 
+* Select either: *Clone to new* or *Clone to existing* (the content will be replaced)
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply modules/ROOT/pages/tutorials/migration-free.adoc:27]: Maybe say that the current content will be overwritten?

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 9]
```diff
@@ -0,0 +1,91 @@
+= Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)
+:description: This section describes migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)
+
+Upgrading your plan to Professional or Enterprise gives you access to additional resources and functionalities to support production workloads and applications with demanding storage and processing needs.
+
+*Migration options*
+
+* Upgrade to Professional (Works for AuraDB)
+* Clone to New (Works for Aura Professional and AuraDS) 
+* Manual Process
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:10]:
```suggestion
* Manual process
```

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+
+Click the ellipsis (*...*) button on an instance card > *Upgrade to Professional*
+
+Verify that the cloud provider and region are correct and select the instance size you need.
+Note that the default version of Neo4j is 5.
+Once you are satisfied, click ***Upgrade***.
+
+== Clone (Works for Aura Professional and AuraDS)
+
+The other way is to _clone_ your existing instance to the Professional tier.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:24]:
```suggestion
The other way is to _clone_ your existing instance to the Professional plan. 
```

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+. (Optional but recommended) Capture existing index and constraint definitions:
+.. Run the following Cypher statement:
++
+[source,cypher]
+----
+SHOW CONSTRAINTS YIELD createStatement
+----
++
+Save result to a file, to use later in the process.
+..
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:43]:
```suggestion
```

File: modules/ROOT/pages/tutorials/migration-free.adoc [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+=== In the console (Activities on AuraDB Professional or AuraDB Enterprise)
+
+. Create a new AuraDB Instance in AuraDB Professional or AuraDB Enterprise with the right resource sizing.
+. Upload via Console drag and drop or push-to-cloud
+.. From the Aura Console: drag and drop
+.. Using the command line: `neo4j-admin push-to-cloud`
+
+=== In the newly created AuraDB Professional or Enterprise Instance
+
+(Optional) Once the AuraDB Instance is loaded and started, you can recreate the indexes and constraints, using the information captured earlier in the process.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/migration-free.adoc:90]:
```suggestion
+
For more information about indexes and constrains, see link:{neo4j-docs-base-uri}/cypher-manual/current/indexes/[Cypher Manual -> Indexes] and link:{neo4j-docs-base-uri}/cypher-manual/current/constraints/[Cypher Manual -> Constraints].
+
. In the console of your existing instance (AuraDB Free), do the following:

.. Download snapshot/Dump locally (the daily automatic snapshot)
.. In the Aura Console select the AuraDB Instance
.. Go to the *Snapshots* tab
.. Click the *three dots*, and select *Export*
.. Save the dump file locally (preserve the .dump extension)
+
. Then create a new AuraDB Instance in AuraDB Professional or AuraDB Enterprise with the right resource sizing.
From your new instance, do the following:

.. Upload via Console drag and drop **or** push-to-cloud
... From the Aura Console: drag and drop the .dump file
... Using the command line: `neo4j-admin push-to-cloud`
+
. In the newly created AuraDB Professional or Enterprise Instance
+
(Optional) Once the AuraDB Instance is loaded and started, you can recreate the indexes and constraints, using the information captured earlier in the process.
```

**Review by AlexicaWright — COMMENTED** [review]
Alright, last batch (I think..)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free Instance to another AuraDB plan (Pro or Enterprise)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

File: modules/ROOT/pages/tutorials/migration-free.adoc (hunk 4 above)
Comment (fiquick) [reply modules/ROOT/pages/tutorials/migration-free.adoc]: I will find out!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free instance to another AuraDB plan (AuraDB Professional or AuraDB Virtual Dedicated Cloud)](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-369.surge.sh

New pages:
- [Migrating your Neo4j AuraDB Free instance to another AuraDB plan](https://neo4j-docs-aura-369.surge.sh/aura/tutorials/migration-free)

**Pull request merged**